AlertingApi.yaml

```diff
@@ -0,0 +1,363 @@
+---
+openapi: 3.0.1
+info:
+  title: FRAIHMWORK Alerting API
+  description: 'FRAIHMWORK allows for notifications via text or email about faults that are affecting components. To enable this, users with phone / email information must be modelled in the system, as well as notification policies that associate users to the notifications they want to receive. The purpose of this API is to add, delete, get or modify these data. This initial version does not imply any protections against malicious behavior from trusted endpoints. This API will versioned according to SemVer 2.0 rules (as described here: https://semver.org/)'
```

Comment:
Also, I'm good with the language in here about notification policy. I'd rather keep it in, since it is coming in the future, and just not expose the endpoint until it is implemented.